i18n/id.json

```diff
@@ -37,7 +37,9 @@
   "I18N_FOOTER_BROWSE_LIBRARY": "Lihat Perpustakaan",
   "I18N_FOOTER_CONTACT_ALL_CAPS": "HUBUNGI KAMI",
   "I18N_FOOTER_CONTACT_US": "Hubungi Kami",
+  "I18N_FOOTER_CONTRIBUTE_ALL_CAPS": "SUMBANG",
```

Comment:
Oh I get it, in that case:
"I18N_FOOTER_DONATE": "Donasi",
"I18N_TOPNAV_DONATE": "Donasi",
"I18N_FOOTER_CONTRIBUTE_ALL_CAPS": "BERKONTRIBUSI",